### PR TITLE
add rule for Bayern.de

### PR DESCRIPTION
**Pull request by numismatika**

### CODE REVIEW

**Comment by rid007** [comment]
It's doesn't work properly with current HTTPS Everywhere browser plugin

**Comment by numismatika** [comment]
@rid007  what exactly is wrong? The map service seems to be fine

**Comment by rid007** [comment]
There is **no ruleset** with the latest version and also in **HTTPS Everywhere Atlas**. [https://www.eff.org/https-everywhere/atlas/?bayern](url)
There is a ruleset for statistik.bayern.de not from `www.bayern.de`
The main problem is they only the HTTPS cert. for `www.bayern.de` not for `bayern.de`.

File: src/chrome/content/rules/Bayern.de.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<!-- cert mismatch on:
+	- www.ldbv.bayern.de
+	- www.gdi.bayern.de/
```
Comment (J0WI) [inline src/chrome/content/rules/Bayern.de.xml]: At least `^` should also be documented.

File: src/chrome/content/rules/Bayern.de.xml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	<target host="www.justiz.bayern.de"/>
+	<target host="www.pki.bayern.de"/>
+	<target host="rvr-prx.bayern.de"/>
+	<target host="intergeo10.bayernwolke.de"/>
+	<target host="intergeo11.bayernwolke.de"/>
+	<target host="intergeo12.bayernwolke.de"/>
+	<target host="intergeo13.bayernwolke.de"/>
+	<target host="intergeo14.bayernwolke.de"/>
+	<target host="intergeo15.bayernwolke.de"/>
+	<target host="intergeo16.bayernwolke.de"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Bayern.de.xml]: Could you try to add more subdomains?

File: src/chrome/content/rules/Bayern.de.xml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<!-- cert mismatch on:
+	- www.ldbv.bayern.de
+	- www.gdi.bayern.de/
+
+	intergeo domains outside of the 10-16 range do not support TLS
+-->
+<ruleset name="Bayern.de">
```
Comment (J0WI) [inline src/chrome/content/rules/Bayern.de.xml]: Please mark the rule as `partial` (depending on how many securable/non-working domains you will find).

**Comment by J0WI** [comment]
Please merge `StatistikamtBayern.xml` into this rule.

**Comment by jeremyn** [comment]
@numismatika Are you still interested in working on this?

**Comment by numismatika** [comment]
It is just a matter of proper splitting, otherwise we will either get the rule disabled  if we go for wildcards or  never be able to merge it because of the large amount of subdomains.

**Comment by jeremyn** [comment]
I recommend making this a `(partial)` ruleset and just including some of the most important subdomains. 1000 is way, way too many. Aim for more like 20-30.

**Pull request closed without merging**